docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -12,6 +12,19 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/react-sdk                   | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-03-14
+
+Released: `@100mslive/hms-video-store@0.9.0`, `@100mslive/react-sdk@0.7.0`
+
+#### Added:
+- `getAuthTokenByRoomCode` in hmsActions to get token for room short code.
+
+#### Fixed:
+- `Uncaught exceptions` are handled properly for most cases.
+
+#### Breaking:
```

Comment:
what we should add in breaking though is - 

- SDK now handles subscribing and unsubscribing to video based on the element's visibility. If you're using `hmsActions.detachVideo` function to unsubscribe to video for e.g. when it's not visible, you may have to pass the flag [`autoManageVideo: false`](<link to render-video section>) in join config. This will avoid any conflict between SDK and your app's code by disabling SDK's functionality.